src/redux/authSlice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface AuthState {
+  isLoggedIn: boolean;
+  currentUserId: number | null
+}
+
+const initialState: AuthState = { 
+  isLoggedIn: false,
+  currentUserId: null,
+};
```

Comment:
You could consider creating separate file for all your interfaces _(e.g. app/interfaces.ts)_ to stick to the principle of 'separation of concerns'. This file would then be more focused on the slicer logic. If you prefer it here, you can keep it here!